docs/tutorials/cpp.md

```diff
@@ -0,0 +1,92 @@
+# Using ICU4X from C++
+
+_We are still working on improving the user experience of using ICU4X from other languages. As such, this tutorial may be a bit sparse, but we are happy to answer questions on our [discussions forum] and help you out _
```

Comment:
Nit: Move this disclaimer down to the bottom of the first section, not the top (just above `## Building and linking ICU4X`). No need to propagate the percention that "the dust hasn't settled" before you say anything else.